client/client_test.go

```diff
@@ -549,7 +595,25 @@ func TestConcurrentIncrements(t *testing.T) {
 		if err := db.DelRange(testUser+"/value-0", testUser+"/value-1x"); err != nil {
 			t.Fatalf("%d: unable to clean up: %v", k, err)
 		}
-		concurrentIncrements(db, t)
+		concurrentIncrements(db, t, false /*isMultiphase*/)
+	}
+}
+
+// TestMultiphaseTransactions is a simple explicit test for serializability
+// while running multiphase trasactions.
+func TestMultiphaseTransaction(t *testing.T) {
+	defer leaktest.AfterTest(t)
+	s := server.StartTestServer(t)
+	defer s.Stop()
+	db := createTestClient(s.ServingAddr())
+
+	// Convenience loop: Crank up this number for testing this
+	// more often. It'll increase test duration though.
+	for k := 0; k < 5; k++ {
+		if err := db.DelRange(testUser+"/value-0", testUser+"/value-1x"); err != nil {
+			t.Fatalf("%d: unable to clean up: %v", k, err)
+		}
+		concurrentIncrements(db, t, true /*isMultiphase*/)
```

Comment:
I don't understand the need for this. Do we really think "multiphase" transactions are going to have different behaviour wrt serializability? Every transaction is already comprised of multiple calls; all we need to test here is that we can go to `[]byte` and back and have the transaction commit successfully and then verify that it wrote some data.